sql/src/test/java/io/crate/operation/aggregation/impl/MaximumAggregationTest.java

```diff
@@ -86,6 +84,6 @@ public void testString() throws Exception {
 
     @Test(expected = NullPointerException.class)
     public void testUnsupportedType() throws Exception {
-        Object[][] result = executeAggregation(DataTypes.OBJECT, new Object[][]{{new Object()}});
```

Comment:
Please avoid unrelated cleanups like this in a feature PR like this. Makes it much more difficult to review